modules/tinymce/src/plugins/table/test/ts/browser/TableSectionApiTest.ts

```diff
@@ -130,6 +156,21 @@ UnitTest.asynctest('browser.tinymce.plugins.table.TableSectionApiTest', (success
 </tbody>
 </table>`;
 
+  const selectAllCells = (type: 'td' | 'th') =>
```

Comment:
This is a chain, but it's missing the naming convention
```suggestion
  const cSelectAllCells = (type: 'td' | 'th') =>
```

- [ ] Fix function name